addons/mrp/models/mrp_workorder.py

```diff
@@ -141,7 +144,10 @@ def _compute_allowed_lots_domain(self):
             finished_workorder_lines = production.workorder_ids.mapped('finished_workorder_line_ids')
             qties_done_per_lot = defaultdict(list)
             for finished_workorder_line in finished_workorder_lines:
-                qties_done_per_lot[finished_workorder_line.lot_id.id].append(finished_workorder_line.qty_done)
+                # It is possible to have finished workorder lines without a lot (eg using the dummy
```

Comment:
@amoyaux you never set qties_done_per_lot anymore?

Reply:
oups